hazelcast/src/main/java/com/hazelcast/cache/impl/CacheEntryViews.java

```diff
@@ -85,13 +86,23 @@ private CacheEntryViews() {
      * @return the {@link LazyCacheEntryView} instance
      */
     public static CacheEntryView<Data, Data> createLazyEntryView(Data key, Data value, Data expiryPolicy, CacheRecord record) {
-        CacheEntryView entryView = new LazyCacheEntryView(key, value,
-                                                          record.getCreationTime(),
-                                                          record.getExpirationTime(),
-                                                          record.getLastAccessTime(),
-                                                          record.getHits(),
-                                                          expiryPolicy);
-        return entryView;
+        return new LazyCacheEntryView<>(key, value,
+                record.getCreationTime(),
+                record.getExpirationTime(),
+                record.getLastAccessTime(),
+                record.getHits(),
+                expiryPolicy);
+    }
+
+    public static <K, V> CacheEntryView<K, V> createLazyEntryView(CacheEntryView<K, V> entryView,
```

Comment:
to be consistent with map side naming, this can be called as `toLazyCacheEntryView`?

Reply:
Updated.